dashboard/app/models/lesson.rb

```diff
@@ -28,10 +28,12 @@ class Lesson < ActiveRecord::Base
   include Rails.application.routes.url_helpers
   include SerializedProperties
 
-  has_many :script_levels, -> {order('position ASC')}, inverse_of: :lesson, foreign_key: 'stage_id'
-  has_one :plc_learning_module, class_name: 'Plc::LearningModule', inverse_of: :lesson, foreign_key: 'stage_id', dependent: :destroy
   belongs_to :script, inverse_of: :lessons
   belongs_to :lesson_group
+  has_many :script_levels, -> {order(:chapter)}, foreign_key: 'stage_id', dependent: :destroy
```

Comment:
do we know why it's safe to remove position from the order here?

Reply:
whoa, weird! TIL.
```
[development] dashboard > Script.find_by_name('valentine').script_levels.map(&:chapter)
=> [1, 2, 3, 4]
```

Reply:
Yup. Another reason we need to clean up these values :)

Reply:
Could you add a comment somewhere in ScriptLevel to explain what chapter is?

Reply:
Added comment